src/ObjCRuntime/Class.cs

```diff
@@ -66,34 +70,37 @@ public Class (Type type)
 			this.handle = GetClassHandle (type);
 		}
 
+#if NET
+		internal Class (IntPtr handle)
+#else
 		public Class (IntPtr handle)
+#endif
 		{
 			this.handle = handle;
 		}
 
 		[Preserve (Conditional = true)]
+#if NET
+		internal Class (IntPtr handle, bool owns)
```

Comment:
Is this no longer available as API?

Reply:
@Therzok I ended up putting the other IntPtr constructor back in.